build/rust_toolchains.bzl

```diff
@@ -0,0 +1,41 @@
+load("@rules_rust//crate_universe:repositories.bzl", "crate_universe_dependencies")
+load("@rules_rust//rust:repositories.bzl", "rules_rust_dependencies", "rust_register_toolchains", "rustfmt_toolchain_repository")
+load("@rules_rust//tools/rust_analyzer:deps.bzl", "rust_analyzer_dependencies")
+load("//deps/rust/crates:crates.bzl", "crate_repositories")
+
+RUST_STABLE_VERSION = "1.86.0"  # LLVM 19
+
+RUST_NIGHTLY_VERSION = "nightly/2025-02-20"
+
+RUST_TARGET_TRIPLES = [
```

Comment:
Nit: rename to RUST_EXTRA_TARGET_TRIPLES to clarify that this is the list of additional triples to be configured on top of the local platform triple, not the list of all platforms we support.